tests/printer/expr/switch.res

```diff
@@ -38,3 +38,14 @@ switch count {
     Js.Console.log("Must be block")
   }
 }
+
+switch route {
+| A => <div> // div tag moves to the next line
```

Comment:
Are there scenarios where we want to keep the ` => <div>` on the same line?

Reply:
divs are behaving as before, only the fragment behavior is changed.

Currently, if there is enough space, everything is actually moved onto a single line:

```rescript
switch route {
| A => <div> <div /> <div /> </div>
| B => <> <div /> <div /> </>
}
```

(which actually isn't very readable either, but that's a separate issue)

Reply:
I wonder if we should move to "smart" line breaking:
1. if the user wrote `<div>` on the same line: break _if it exceeds line-width_, otherwise keep on the same line.
2. if the user wrote `<div>` on the next line: break by default